src/lib.rs

```diff
@@ -52,11 +52,11 @@ pub struct Lib {
 }
 
 /// Contains information about loaded libraries and also tracks search paths and reloading events.
-pub struct DynamicReload<'a> {
```

Comment:
I am happy to see that lifetime being removed! :+1:

Reply:
nice! I guess I had the lifetime for storing some ref :)